core/core.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     api commonDependencies.'log4j'
     api commonDependencies.'log4j-api'
     api commonDependencies.'swagger-annotations'
+    api 'org.json:json:20190722'
```

Comment:
This dependency and JsonUtils seems unnecessary, in core at least.

Reply:
Yeah, we can move it